rdf_diff_store/git.py

```diff
@@ -31,6 +31,20 @@ def graph_path(repo: Repo, filename: str) -> str:
     return os.path.join(str(repo.working_tree_dir), filename)
 
 
+def delete_graph(id: str) -> None:
+    """Delete graph."""
+    repo = get_repo()
+    filename = graph_filename(id)
+    path = graph_path(repo, filename)
+
+    try:
+        os.remove(path)
+        repo.index.remove([filename])
+        repo.index.commit(f"delete: {id}")
+    except FileNotFoundError:
+        pass
```

Comment:
What? Det er vel god skikk å returnere 404 på Not Found?

Reply:
Om du ønker å slette noe, har det da noe å si at det ikke finnes? Det oppstår ingen problemer, og resultatet er jo uansett at det er borte :shrug:

Reply:
Men joa, flere som er enige i tankegangen: `rm: cannot remove 'foo': No such file or directory`

Reply:
Gir 404 nå! https://github.com/Informasjonsforvaltning/rdf-diff-store/pull/8/commits/486183afd5257548da5f6f5bcddb8fc6bb0e9389